source/developers/sts-for-operator.rst

```diff
@@ -0,0 +1,170 @@
+.. _minio-sts-operator:
+
+===============================================
+Security Token Service (STS) for MinIO Operator
+===============================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+.. versionadded:: Operator v5.0.0
+
+   The MinIO Operator supports a set of API calls that allows an application to obtain STS credentials for a MinIO Tenant.
+
+Benefits of STS for MinIO Operator include:
+
+- :ref:`STS credentials <minio-security-token-service>` allow an application to access objects on a MinIO Tenant without the need to create credentials for the application on the tenant.
+
+- Allows applications to access objects in MinIO tenants using a Kubernetes-native authentication mechanism.
+  
+  Service Accounts or Service Account Tokens are a core concept of :kube-docs:`Role-Based Access Control (RBAC) <reference/access-authn-authz/rbac/>` :kube-docs:`authentication <reference/access-authn-authz/authentication/#service-account-tokens>` in Kubernetes.
+
+- Implementing STS for MinIO Operator allows you to utilize infrastructure as code principles and configuration by using the tenant custom resource definition (CRD) and a MinIO PolicyBinding CRD.
+
+.. important:: 
+
+   MinIO Operator launches with STS *disabled* by default.
+   To use STS with the Operator, you must first explicitly enable it.
+
+How STS Authorization Works in Kubernetes
+-----------------------------------------
+
+An application can use an ``AssumeRoleWithWebIdentity`` call to send a request for temporary credentials to the MinIO Operator.
+The Operator checks the validity of the request, retrieves policies for the application, obtains credentials from the tenant, and then passes the credentials back the application.
+The application uses the issued credentials to work with the object storage on the tenant.
+
+.. image:: /images/k8s/sts-diagram.png
+   :width: 600px
+   :alt: A diagram showing STS token process flow on a Kubernetes MinIO deployment between the requesting application, MinIO Operator, Kubernetes API, PolicyBinding custom resource definition, and the MinIO tenant.
+   :align: center
+
+The complete process includes the following steps:
+
+1. An application sends an API request to the MinIO Operator containing the name of the application and a service account to use.
```

Comment:
Or maybe this is where we need to clarify - from what I'm gathering from Pedro's comments, it's the same AssumeRoleWithWebToken API, but the JWT is the ServiceAccount token for the target tenant.

Reply:
I mentioned this earlier on the page now, so leaving it out here for now.